tests/extract_tests.ml

```diff
@@ -8,7 +8,7 @@ let protect ~finally f =
       finally ();
       r
 
-let disabled = [ 206; 215; 216; 519; 539 ]
+let disabled = [ 206; 215; 216; 519 ]
```

Comment:
Outstanding progress. You're demolishing these deviations. Thanks so much @tatchi! :tada: